docs-ref-services/latest/cosmos-db.md

```diff
@@ -28,7 +28,7 @@ npm install @azure/arm-cosmosdb
 ## Samples
 
 * Examples for using this module in Node.js as well as browser applications can be found in the [README for the module](https://www.npmjs.com/package/@azure/arm-cosmosdb)
-* [Developing a JavaScript app using Azure Cosmos DB](https://azure.microsoft.com/resources/samples/azure-cosmos-db-documentdb-nodejs-getting-started/)
+* [Developing a JavaScript app using Azure Cosmos DB](https://docs.microsoft.com/en-us/samples/azure-samples/azure-cosmos-db-sql-api-nodejs-getting-started/azure-cosmos-db-sql-api-nodejs-getting-started/)
```

Comment:
We have guidance to not use locale in links so that it defaults to the right one in the right places.
```suggestion
* [Developing a JavaScript app using Azure Cosmos DB](https://docs.microsoft.com/samples/azure-samples/azure-cosmos-db-sql-api-nodejs-getting-started/azure-cosmos-db-sql-api-nodejs-getting-started/)
```

@sima-zhu Any idea why the checks on the PR didnt catch this?

Reply:
smart!